core/types/transaction.go

```diff
@@ -382,6 +382,15 @@ func (tx *Transaction) Cost() *big.Int {
 	return total
 }
 
+// Cost returns (gas * gasPrice) + (blobGas * blobGasPrice).
+func (tx *Transaction) GasCost() *big.Int {
```

Comment:
I think this is more neat: `return new(big.Int).Sub(tx.Cost(), tx.Value())`, what do you think ?